npm/janus.d.ts

```diff
@@ -127,6 +127,9 @@ declare namespace JanusJS {
 		handleRemoteJsep(params: { jsep: JSEP }): void;
 		dtmf(params: any): void;
 		data(params: any): void;
+		isAudioMuted(): boolean;
+		muteAudio(): void;
+		unmuteAudio(): void;
```

Comment:
This is the reason why this pr was created. And I also noticed that adapter has `any` type and I thought it's ok to get actual types from webrtc-adapter.